pytorch/Dockerfile

```diff
@@ -109,6 +143,14 @@ RUN wget -q  --no-check-certificate https://raw.githubusercontent.com/oneapi-src
     wget -q  --no-check-certificate https://raw.githubusercontent.com/intel/neural-compressor/master/docker/third-party-programs-pytorch.txt -O /licensing/third-party-programs-pytorch.txt && \
     wget -q  --no-check-certificate https://raw.githubusercontent.com/intel/neural-compressor/master/LICENSE -O /licensing/LICENSE
 
+ENV BASH_ENV='$HOME/.bash_startup'
+
+FROM multinode AS hf-genai
+
+COPY hf-genai-requirements.txt .
+
+RUN python -m pip install --no-cache-dir -r hf-genai-requirements.txt
+
```

Comment:
As we discussed on slack, this will need the `ENV` defined again, otherwise I'm getting errors when using the CCL backend from k8s:
```
ENV I_MPI_ROOT=${I_MPI_ROOT}
ENV CCL_ROOT=${CCL_ROOT}
ENV FI_PROVIDER_PATH=${FI_PROVIDER_PATH}
ENV LIBRARY_PATH=${LIBRARY_PATH}
ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}
ENV PATH=${PATH}
ENV CPATH=${CPATH}
```

Reply:
You can remove these `ENV`s. Using the entrypoint from the multinode container and changing the PyTorchJob spec to use [args instead of command](https://github.com/intel/ai-containers/pull/203/files#diff-57aa7e05f2767ef0d092df5f5ff0b30dc920cfd56cd05131078321273adab4dd) worked.